components/inputs/inputs.css

```diff
@@ -0,0 +1,45 @@
+.input-group {
+    color: #333;
+    font-size: 0.8rem;
+    line-height: 1.2rem;
+    margin: 0 1.2rem 0.8rem;
+    width: 20%;
+}
+  
+label {
+    display: block;
+    margin-bottom: 2px;
+}
+
+input[type=text] {
+    background: #fff;
+    border: 1px solid #999;
+    padding:0.6rem 0.7rem;
+    float: left;
+    font-size: 0.8rem;
+    margin: 5px 0;
+    width: 100%;
+    transition: 0.3s all ease;
+    outline: none;
+    border-radius: 2px;
+}
+
+.input-group input:focus{
+    border : 1px solid #1FABD7;
+}
+
+.input-group.err label{
+    color: #F44336;
+}
+
+.input-group.err input{
+    border: 1px solid #F44336;
+}
+
+.input-group.err input:focus{
+    border: 1px solid #F44336;
+}
+
+.error-message{
+    color: #F44336;
+}
```

Comment:
Add a new line at the end of the file.
Reference: [link](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)

Reply:
Got it will keep this in mind from next PR